src/server.ts

```diff
@@ -44,6 +45,33 @@ export async function handler(req: IncomingMessage, res: ServerResponse) {
             res.setHeader('Content-Type', mimeType(pathname));
             res.setHeader('Cache-Control', cacheControl(isProd, cacheResult ? 7 : 0));
             res.end(JSON.stringify(result));
+        } else if (pathname === pages.compare) {
+            let data: any[] = [];
+            req.on('data', chunk => data.push(chunk));
+            req.on('end', () => {
+                const [packageString] = data.toString().match(/{[\s\S]+}/) || [];
+                if (!packageString) {
+                    return renderPage(res, '/parse-failure', query, TMPDIR, GA_ID);
+                }
+                let packageData;
```

Comment:
```suggestion
                let packageData: PackageJson;
```

Let's add a type here

Reply:
Good idea, done. 👍